source/dump.f90

```diff
@@ -429,9 +426,10 @@ subroutine dump_initialise
 
   implicit none
 
-  integer i,j
-  logical lopen
+  integer i,j,k,l
+  logical lOpen, rErr
   character(len=256) filename
+  character(len=16) tasbuf(6,6)
```

Comment:
Can you guarantee that len=16 is enough?

Reply:
Yes. That is determined by the call to the casting routine.